src/test/java/org/kairosdb/datastore/cassandra/CassandraDatastoreTest.java

```diff
@@ -182,7 +183,7 @@ public static void setupDatastore() throws InterruptedException, DatastoreExcept
 
         // TODO: test the caches being hit
         final StringKeyCache stringCache = mock(StringKeyCache.class);
-        s_datastore = new CassandraDatastore(new CassandraClientImpl(cassandraConfig), cassandraConfig,
+        s_datastore = new CassandraDatastore(new CassandraClientImpl(cassandraConfig, GlobalTracer.get()), cassandraConfig,
```

Comment:
You could directly provide `NoopTracer` here to make sure that `GlobalTracer.get()` won't brake anything by occasion

Reply:
I guess Globaltracer.get() will return NoopTracer until a tracer is registered. Is that correct?
http://static.javadoc.io/io.opentracing/opentracing-util/0.31.0/io/opentracing/util/GlobalTracer.html#get--